distributed/nanny.py

```diff
@@ -151,8 +151,7 @@ def _kill(self, comm=None, timeout=10):
             except CommClosedError:
                 pass
             except BaseException as e:
-                if (not self.loop._closing and self.loop._running and not
-                        self.loop._stopped):
+                if self.loop._running:
```

Comment:
I hope this is ok, I'm not sure why the "if" had three checks beforehand. The AioLoop shim only supports "_running", and it seems better to only check "_running" here, rather than add shims for "_closing" and "_stopped".